src/terraform/install.sh

```diff
@@ -24,7 +24,7 @@ TERRAGRUNT_SHA256="${TERRAGRUNT_SHA256:-"automatic"}"
 TFSEC_SHA256="${TFSEC_SHA256:-"automatic"}"
 TERRAFORM_DOCS_SHA256="${TERRAFORM_DOCS_SHA256:-"automatic"}"
 
-TERRAFORM_GPG_KEY="72D7468F"
+TERRAFORM_GPG_KEY="34365D9472D7468F"
```

Comment:
Can you point me to the doc which mentions about this key?
According to https://www.hashicorp.com/security the current key looks fine.

Reply:
The key is part of the fingerprint at `PGP Public Keys` (the last 64-bit).
I tested to find the key manually but this didn't worked with the 32-bit version for the used keyserver:

- [keyserver.ubuntu.com](https://keyserver.ubuntu.com/pks/lookup?search=72D7468F&fingerprint=on&op=index)
- [keys.openpgp.org](https://keys.openpgp.org/search?q=72D7468F)
- [keyserver.pgp.com](https://keyserver.pgp.com/vkd/SubmitSearch.event?SearchCriteria=72D7468F)

And the same happened at the OS level. The only keyserver where I was able to find the Public Key with the 32-bit version is [keybase.io](https://keybase.io/hashicorp).

Reply:
Just noticed these keys are both suffixes of the fingerprint. (https://github.com/devcontainers/features/issues/323#issuecomment-1348151198)